debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -133,10 +134,15 @@ protected void initPublication() {
         if (PostgresConnectorConfig.LogicalDecoder.PGOUTPUT.equals(plugin)) {
             LOGGER.info("Initializing PgOutput logical decoder publication");
             try {
+                // Unless the autocommit is disabled the SELECT publication query will stay running
+                Connection conn = pgConnection();
+                conn.setAutoCommit(false);
```

Comment:
Can't we even generally disable auto-commit, right when creating the connection? I don't see when we ever would want it to be enabled?

Reply:
Ok, gotcha. Agreed. It "should be fine (TM)", but, yeah...